### ISSUE TITLE
Feature request: new rat reference mRatBN7.2

### ISSUE THREAD

**Issue by klkeys** [ewels/AWS-iGenomes#15]
any interest in adding the most recent rat reference?

https://rgd.mcw.edu/wg/news/new-rat-genome/

happy to contribute where I can; I've never added to iGenomes before

**Comment by ewels** [comment]
Hi @klkeys,

Thanks for the interest and offer of help! I haven't added any new references to iGenomes for a long time now (years) because it's quite a lot of very manual work. It's a lot more than just downloading and hosting the files - there's a bunch of normalisation that needs to be done to normalise chromosome names and all sorts (see the illumina iGenomes page: https://emea.support.illumina.com/sequencing/sequencing_software/igenome.html)

I had hoped that illumina would continue to maintain iGenomes and I could mirror that, but unfortunately it seems to be abandoned.

Not all is lost though, for some time now I have been looking towards RefGenie as a successor to AWS-iGenomes: http://refgenie.databio.org/ - it was created for exactly the same reason as AWS-iGenomes but is built on a back end that makes maintenance far easier. It's now also available on AWS with the registry of open data: https://registry.opendata.aws/refgenie/ (see also http://refgenomes.databio.org/ )

We're hoping to mirror over everything from AWS-iGenomes to RefGenie and start using that instead for @nf-core (see issue: https://github.com/nf-core/tools/issues/1086, this has stalled a bit but should hopefully soon be picked up again).

So in short - I'm hoping that AWS-iGenomes will stay online for a long time to come (mostly for backwards compatibility of code that relies on it), but I'm not super keen on adding new stuff to it if I can avoid it. I hope this is ok..

Shout if you have any questions!

Phil
